examples/generalized_linear_models/GLM-truncated-censored-regression.ipynb

```diff
@@ -0,0 +1,1356 @@
+{
```

Comment:
<ul><li>Same comments as for the first model</li><li>Out of curiosity, can there be cases where you don't know the exact bounds? What can you do then? Model the bounds as well?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>I think the bounds can be random variables. Stan allows for it, not sure how well our TruncatedNormal accommodated them though </p>

Reply:
<p>I can add some info to say that this is doable. In practice I'm not sure how useful it is to treat the bounds as a latent variable though and requires domain knowledge to formulate a prior for what the bounds are. My feeling is that this is too niche for this example notebook.</p>

Reply:
<p>Yeah, no need to add that in the NB if it's too niche -- I was just curious ;)</p>